app/src/components/campgrounds/AddCampground.vue

```diff
@@ -1,25 +1,75 @@
 <template>
-  <form @submit.prevent="handleSubmit">
+  <form @submit.prevent="onAdd(campground)">
     <input v-model="campground.name" require>
```

Comment:
```suggestion
    <input v-model="campground.name" required>
```
Missing a "d" on required. I think this may be the syntax error that was originally throwing Travis.